style.css

```diff
@@ -87,7 +87,7 @@
 /* today */
 #works-container {
   display: grid;
-  grid-template-rows: 150px auto 450px 450px 450px 100px;
+  grid-template-rows: 150px auto 450px 450px 450px 450px 450px 450px 100px;
   padding-top: 8em;
 }
```

Comment:
- [ ] Impressive styling for your works. Also, the missing part here is the `hover` effect. Please make sure there is a `see project` button that appears on hover for each project. when the project is hovered on with the mouse the description and dark opacity disappear and a `See Project` button appears to the bottom of the image and takes the whole image width.

### Figma design `on hover`
![card-hover](https://user-images.githubusercontent.com/69077061/206689801-e6733930-3ded-4aa9-b50f-fd0c54098b32.PNG)

### Your design
![card](https://user-images.githubusercontent.com/69077061/206689684-213c2ea8-ae99-48e9-a36f-48ffeb86a029.PNG)